### PR TITLE
Add new habit/#1000

### PR DESCRIPTION
**Pull request by romanovskyroma**
create "add-new-habit" component
[#1785](https://github.com/ita-social-projects/GreenCity/issues/1785)
new-habit

### CODE REVIEW

**Review by JimHolman1199 — COMMENTED** [review]
Check unit tests

**Comment by romanovskyroma** [comment]
> Check unit tests

add '_' to spec.ts

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/component/shared/components/calendar-base/calendar-image.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const calendarImage = {
```
Comment (luchward) [inline src/app/component/shared/components/calendar-base/calendar-image.ts:1]: What is the type of this file? Why did you add separate file for the constant?

**Review by luchward — APPROVED** [review]

**Review by Benerous — APPROVED** [review]

**Review by skrebetsnataliia — APPROVED** [review]

**Review by vitaliipolitylo — APPROVED** [review]
